locales/en-US/messages.properties

```diff
@@ -91,3 +91,52 @@ i_donated_to_thunderbird=I donated to @mozthunderbird today to #freetheinbox. Jo
 
 # Site title when shared on Facebook
 support_mozilla=Support Mozilla
+
+# faq page
+faq_header=FAQ
+faq_donate_link=CLICK HERE TO DONATE NOW
+faq_intro=Below are the most frequently asked questions about donating to the Mozilla Foundation. If your question and answer are not listed here, please contact us at <a href="mailto:donate@mozilla.org">donate@mozilla.org</a>.
+faq_item_1_header=What are the ways I can donate?
+faq_item_1_paragraph=<a href="https://donate.mozilla.org/">Visit this link to make a secure online donation right now via credit card or PayPal</a>. For additional ways to give -- like check, bitcoin, or bank transfer -- visit our <a href="https://wiki.mozilla.org/Ways_to_Give"Ways to Give page</a>.
+faq_item_2_header=How does Mozilla ensure my personal information is protected?
+faq_item_2_paragraph_a=We care about the privacy of our donors and take our <a href="https://www.mozilla.org/en-US/privacy/principles/">Privacy Principles</a> seriously. Mozilla believes individuals should be in control of their data, and maintains multi-layered security controls and practices. To learn more, read the <a href="https://www.mozilla.org/en-US/privacy/">Mozilla Privacy Policy</a>.
+faq_item_2_paragraph_b=In certain instances, when you donate to Mozilla via a third party like PayPal or Stripe, those organizations also collect user data.
+faq_item_3_header=Why do you need my address in order to process a donation?
+faq_item_3_paragraph=We understand that your privacy is very important. We ask for a minimum amount of information required to process credit card donations, including billing addresses. This allows our payment processor to verify your identity, process your payment, and prevent fraudulent charges to your credit card. We keep your information private -- if you have questions, please refer to our <a href="https://www.mozilla.org/en-US/privacy/principles/">Privacy Policy</a>. If you would rather not fill in your information on our online donation form, you can <a href="https://wiki.mozilla.org/Ways_to_Give#Check_.28via_postal_service.29">mail a check</a> or donate directly <a href="https://sendto.mozilla.org/page/contribute/Give-Now">via Paypal</a>.
+faq_item_4_header=Where do I send a check/cheque?
+faq_item_4_paragraph_a=Please send your check to:
+faq_item_4_paragraph_b=Mozilla Foundation
```

Comment:
We should actually hardcode that and the rest of the address, we don’t want people to change anything to this US address